src/doc/pythonbindings.rst

```diff
@@ -35,6 +35,38 @@ of brevity, we will alias the package name as follows:
 
 .. _sec-pythontypedesc:
 
+
+Jupyter Notebooks
+=================
+
+Like any other Python package, OpenImageIO can be used in `Jupyter notebooks <https://jupyter.org/install>`_. 
+The ImageBuf objects support getting displayed inline within notebooks.
+
```

Comment:
This is a very odd place for this discussion to go -- it's literally inside the section that's talking about the TypeDesc class.

I would recommend that in the ImageBuf section of this chapter, you put a brief description for the `_repr_png_` method itself to document its presence. 

Then move this part you've written here, which is really a "how to" for incorporating OIIO into Jupyter Notebooks and viewing the results, to the *very end* of the chapter, because it actually fits well in the section called "Python Recipes".

Reply:
The text of what you've written here is great. I just think you should move it to the end of the "Recipes" section.